### PR TITLE
basic Start screen

### PR DESCRIPTION
**Pull request by MeKamm74**
looks pretty garbage but it works
Start launches the SampleScene
Quit runs Application.Quit(), which only works when the project is built, so it also logs a debug thing.

### CODE REVIEW

**Comment by MattCharles** [comment]
+1

File: Assets/Scripts/MainMenu.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public class MainMenu : MonoBehaviour {
+
+	public void StartGame () 
+	{
+		Debug.Log("STARTING GAME");
+		SceneManager.LoadScene (SceneManager.GetActiveScene ().buildIndex + 1);
+	}
+
+	public void QuitGame () 
+	{
```
Comment (RyanCross) [inline Assets/Scripts/MainMenu.cs]: if you add the following property IF statement here, this logic will work in the editor 

```
  #if UNITY_EDITOR
         // Application.Quit() does not work in the editor so
         // UnityEditor.EditorApplication.isPlaying need to be set to false to end the game
         UnityEditor.EditorApplication.isPlaying = false;
 #else
         Application.Quit();
  #endif
```

File: Assets/Scripts/GameController.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 public class GameController : MonoBehaviour {
     PlayerController player;
-    TrackGenerationController trackGenerationController;
+    // TrackGenerationController trackGenerationController;
```
Comment (RyanCross) [inline Assets/Scripts/GameController.cs]: This class is in master now so we can leave this uncommented.

**Comment by RyanCross** [comment]
Also make sure that: 
All prefabs have the latest settings from the scene
The scene has been saved (it doesn't auto do this)

**Pull request merged**